_jekyll/includes/2-base/header.html

```diff
@@ -4,6 +4,6 @@
 
 
 <header>
-    <div class="header-image"></div>
+    <a href="/" class="header-image"></a>
```

Comment:
You might want to change this to 
```
<a class="header-image" href="/" aria-label="Home" aria-hidden="true" tabindex="-1"></a>
```
The purposes of them are:
aria-label : just to get around the automated testing that would complain about empty anchor link
aria-hidden and tabindex : so that it is not focusable and removed from the accessibility tree. This way the visually-able mouse user can always click on it to go to the home page, the visually-impaired user won't head the "Home" button twice, and the visually-able-keyboard user don't get confused when they don't know where the mouse focus is.